test/extended/workspacetype/workspace.go

```diff
@@ -61,4 +62,67 @@ var _ = g.Describe("[sig-workspace]", func() {
 			return workSpaces
 		}, 60*time.Second, 5*time.Second).ShouldNot(o.ContainSubstring(myWorkSpace.Name))
 	})
+
+	// author: zxiao@redhat.com
+	g.It("Author:zxiao-Medium-I can list child workspaces under parent workspace [Smoke]", func() {
+		g.By("1) Create a test workspace as parent workspace")
+		k.SetupWorkSpace()
+		parentWorkSpace := k.WorkSpace()
+		defer func() {
+			output, err := k.WithoutNamespace().WithoutKubeconf().WithoutWorkSpaceServer().Run("delete").Args("--server="+parentWorkSpace.ParentServerURL, "workspace", parentWorkSpace.Name).Output()
+			o.Expect(err).NotTo(o.HaveOccurred())
+			o.Expect(output).To(o.ContainSubstring("deleted"))
+		}()
+
+		g.By("2) Create five child workspaces under the parent workspace")
+		childWorkSpaces := []exutil.WorkSpace{}
+		for i := 1; i <= 5; i++ {
+			g.By(fmt.Sprintf("2.%v) Create child workspace no.%v", i, i))
+			k.SetupWorkSpaceWithSpecificPath(parentWorkSpace.ServerURL)
+			childWorkSpace := k.WorkSpace()
+			childWorkSpaces = append(childWorkSpaces, childWorkSpace)
+		}
+
+		g.By("3) List child workspaces under the parent workspace, check if all workspaces are correctly displayed")
+		workSpaces := k.ListWorkSpacesWithSpecificPath(parentWorkSpace.ServerURL)
+		for i := 0; i < len(childWorkSpaces); i++ {
+			ws := childWorkSpaces[i]
+			o.Expect(workSpaces).To(o.ContainElement(ws.Name))
+		}
+	})
+
+	// author: zxiao@redhat.com
+	g.It("Author:zxiao-Medium-I can create context for a specific workspace and use it [Smoke][Serial]", func() {
```

Comment:
Seems we'd better unify differenf lables at different location. `[Smoke]` Maybe add before case name is better. We could wait other folks' suggestions.